GeneratorInterface/LHEInterface/src/LHERunInfo.cc

```diff
@@ -346,18 +345,17 @@ void LHERunInfo::statistics() const
 		  break;
 		}
 
-		if(fracAcc<1e-6)continue;
 
 		double fracBr = proc->accepted().sum() > 0.0 ?
 		                proc->acceptedBr().sum() / proc->accepted().sum() : 1;
-		double sigmaFin = sigmaAvg * fracAcc;
+		double sigmaFin = fracAcc < 1e-6? 0: sigmaAvg * fracAcc;
```

Comment:
It's an old habit of mine whenever comparing two numbers. 
I will change it to 0 since the cross sections could be much smaller than 1e-6 for some exotica process.